index.html

```diff
@@ -1,116 +1,176 @@
+                    <h2 class=visually-hidden>Приемущества
+                    </h2>
+            
+            <ul class="features-list">
+                <li class="feature-item">
+                    <h3>Быстро
+                    </h3>
+                    <p>Мы делаем свою работу быстро! Два часа пролетят незаметно и Вы - счастливый обладатель стильной стрижки-минутки!
+                    </p>
+                </li>
+                
+                <li class="feature-item">
+                    <h3>Круто
+                    </h3>
+                    <p>Забудьте, как Вы стриглись раньше. Мы селаем из Вас звезду футбола или кино! Во всяком случае внешне.                        
+                    </p>
             
-        <li class="feature-item">
-            <h3>Дорого</h3>
-            <p>Наши мастера - профессионалы своего дела и не могут стоить дешево.К тому же, разве цена не дает определенный статус?</p>
-        </li>
-             </li>
-    </ul>
-</section>
-<section class="news">
-    <h2>Новости</h2>
-    <ul class="news-list">
-        <li class="news-item">
-            <p>Нам наконец завезли Ягермейстер! Теперь Вы сможете пропустить стаканчик во время стрижки</p>
-            <time datetime="2021-01-28">28 января</time>
-        </li>
-        <li class="news-item">
-            <p>В нашей команде пополнение Борис "ПуджХукер228" Стригунец, обладатель множества титулов и наград пополнил наши стройные ряды</p>
-            <time datetime="2021-01-28">28 января</time>
-        </li>
-    </ul>
-    <a class="button" href="news.html">Все новости о чоуках на чике</a>
-</section>
-<section class="gallery">
-    <h2>Фотогалерея</h2>
-    <figure class="gallery-section">
-        <a href="хз какая ссылка будет"><src="img/studio.jpg" width="286" height="164" alt="Интерьер"></src></a>
-    </figure>
-    <span class="button gallery-button gallery-button-back">Назад</span>
-    <span class="button gallery-button gallery-button-next">Рузке Вперде</span>
-</section>
-<section class="contacts">
-    <h2>Контактная информация</h2>
-    <p>Тут инфа<br>
-    ещё инфа<br>
-    инфа<br>
-    инфа<br>
-    инфа
+                <li class="feature-item">
+                    <h3>Дорого
+                    </h3>
+                    <p>Наши мастера - профессионалы своего дела и не могут стоить дешево.К тому же, разве цена не дает определенный статус?
+                    </p>
+                </li>
+                </li>
+            </ul>
+        </section>
+
+        <section class="news">
+                    <h2>Новости
+                    </h2>
+            <ul class="news-list">
+                <li class="news-item">
+                    <p>Нам наконец завезли Ягермейстер! Теперь Вы сможете пропустить стаканчик во время стрижки                        
+                    </p>
+                        <time datetime="2021-01-28">28 января                            
+                        </time>
```

Comment:
у них один уровень вложенности тоже
```suggestion
                    <p>Нам наконец завезли Ягермейстер! Теперь Вы сможете пропустить стаканчик во время стрижки                        
                    </p>
                    <time datetime="2021-01-28">28 января                            
                    </time>
```